opre_ops/ops_site/models.py

```diff
@@ -30,46 +36,65 @@ class Meta:
 
     def display_name(self):
         return self.first_name + " " + self.last_name
+
     display_name.short_description = "Full name"
     full_name = property(display_name)
 
     def __str__(self):
         return self.full_name
 
 
-class CANInfo(models.Model):
+class CommonAccountingNumber(models.Model):
     """
     A CAN is a Common Accounting Number, which is
     used to track money coming into OPRE
 
-    The CANInfo model contains all the relevant
+    This model contains all the relevant
     descriptive information about a given CAN
     """
 
-    ARRANGEMENT_TYPES = [('OPRE Appropriation', 'OPRE Appropriation'), ('Cost Share', 'Cost Share'), 
-                        ('IAA', 'IAA'), ('IDDA', 'IDDA'), ('MOU', 'MOU')]
+    ARRANGEMENT_TYPES = [
+        ("OPRE Appropriation", "OPRE Appropriation"),
+        ("Cost Share", "Cost Share"),
+        ("IAA", "IAA"),
+        ("IDDA", "IDDA"),
+        ("MOU", "MOU"),
+    ]
 
     number = models.CharField(max_length=30)
     description = models.CharField(max_length=100)
     purpose = models.TextField(default="", blank=True)
     nickname = models.CharField(max_length=30)
     arrangement_type = models.CharField(max_length=30, choices=ARRANGEMENT_TYPES)
     funding_source = models.ManyToManyField(FundingPartner)
-    authorizer = models.ForeignKey(FundingPartner, on_delete=models.PROTECT, related_name="authorizer")
+    authorizer = models.ForeignKey(
+        FundingPartner, on_delete=models.PROTECT, related_name="authorizer"
+    )
+
+    def info_for_fiscal_year(self, fy):
+        return CANFiscalYear.objects.filter(can=self.id, fiscal_year=fy).first()
+
+    def contracts_for_fiscal_year(self, fy):
+        cid = [
+            li.fiscal_year.line_item.contract.id
+            for li in self.line_items_fy.filter(fiscal_year__fiscal_year=fy)
+        ]
+        return Contract.objects.filter(id__in=cid)
 
     class Meta:
-        verbose_name_plural = "CANs Info"
```

Comment:
Seeing this makes me so glad we are getting rid of the name `CAN Info`

Reply:
Re: the README, I don't think so because it shouldn't matter for getting up and running or anything, but it should definitely be documented somewhere! Incoming commit that adds comments to the `docker-compose` file!